.github/workflows/build.yml

```diff
@@ -1,32 +1,41 @@
 name: Build
 
 on:
-    push: ~
+    push:
+        branches-ignore:
+            - 'dependabot/**'
     pull_request: ~
     release:
         types: [created]
     schedule:
         -
             cron: "0 1 * * 6" # Run at 1am every Saturday
+    workflow_dispatch: ~
 
 jobs:
     tests:
-        runs-on: ubuntu-latest
+        runs-on: ubuntu-18.04
 
-        name: "PHP ${{ matrix.php }}, MySQL ${{ matrix.mysql }}, Sylius ${{matrix.sylius}}"
+        name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}"
 
         strategy:
             fail-fast: false
             matrix:
-                php: [7.4, 7.3]
-                node: [10.x]
-                mysql: [5.7, 8.0]
-                sylius: [~1.8, ~1.9]
+                php: ["8.0", "7.4"]
+                symfony: ["^4.4", "^5.2"]
+                sylius: ["~1.9.0", "~1.10.0"]
+                node: ["10.x"]
+                mysql: ["8.0"]
 
                 exclude:
-                    - # PHP 7.3 does not support "caching_sha2_password" authentication plugin which is a default one in MySQL 8.0
-                        php: 7.3
-                        mysql: 8.0
+                    - sylius: ~1.9.0
+                      php: 8.0
+
+                    - sylius: ~1.10.0
+                      symfony: ^4.4
+                    # friendsofsymfony/elastica-bundle - package is locking php 7.x
+                    - sylius: ~1.10.0
```

Comment:
Shouldn't we run tests also for 1.10 with php 8.0?

Reply:
unforunetly we cant use php 8.0 because package "friendsofsymfony/elastica-bundle": "^5.0", requires PHP 7.x
and 6.0 of this package currently is BETA-version.

support for Php 8.0 should be added separately